linux-mint-setup.sh

```diff
@@ -181,6 +181,10 @@ rm postman.tar.gz
 
 # Packet sniffing / viewing tool, install silently.
 sudo DEBIAN_FRONTEND=noninteractive apt-get -y install wireshark
```

Comment:
Shouldn't this line be removed after your changes?

Reply:
No, it should be there because it installs wireshark. I just added configuration.